verticapy/learn/model_selection.py

```diff
@@ -362,42 +368,70 @@ def cross_validate(
                     result["{}-fold".format(i + 1)] = estimator.classification_report(
                         labels=[pos_label], cutoff=cutoff
                     ).values["value"][0:-1]
-                    if (training_score):
+                    if training_score:
                         estimator.test_relation = estimator.input_relation
-                        result_train["{}-fold".format(i + 1)] = estimator.classification_report(
+                        result_train[
+                            "{}-fold".format(i + 1)
+                        ] = estimator.classification_report(
                             labels=[pos_label], cutoff=cutoff
-                        ).values["value"][0:-1]
+                        ).values[
+                            "value"
+                        ][
+                            0:-1
+                        ]
```

Comment:
I don't really see how the new format is better!

Reply:
Indeed, it is the API black which is formatting this way when the code line is too long...